rpcserver.go

```diff
@@ -754,6 +755,11 @@ func (r *rpcServer) addDeps(s *server, macService *macaroons.Service,
 		},
 		SetChannelAuto:     s.chanStatusMgr.RequestAuto,
 		UseStatusInitiated: subServerCgs.RouterRPC.UseStatusInitiated,
+		SetExperimentalEndorsement: func() bool {
+			return clock.NewDefaultClock().Now().Before(
```

Comment:
Same, I'm not sure if the check expressed here is correct to bound `endorsement` until end of the experiment. Looking on the `go`'s `time` package `Time` here sounds to be current local time: https://github.com/golang/go/blob/94aeedc33a4434f0d410be7c42e0bbcdaedd790f/src/time/time.go#L1144("Now returns the current local time").

Note all the distinctions made between monotonic clock reading, wall clock reading and the location associated with the time: https://github.com/golang/go/blob/master/src/time/time.go#L116 and https://github.com/golang/go/blob/94aeedc33a4434f0d410be7c42e0bbcdaedd790f/src/time/time.go#L12